nix/cardano-services/deployments.nix

```diff
@@ -24,6 +24,14 @@ in {
         }
       ];
     };
+    handle-provider-deployment = {
+      spec.template.spec.containers = dmerge.updateOn "name" [
+        {
+          name = "handle-provider";
+          image = cell.oci-images.cardano-services.image.name;
```

Comment:
What's the default this overrides?

Reply:
While diff is not very clear but here is the difference, default is invalid image name to avoid it failing silently

Diff:
```diff
-        image: FIXME-ADD-LATEST-IMAGE-REFERENCE
+        image: 926093910549.dkr.ecr.us-east-1.amazonaws.com/cardano-services:vh1wcqh4y1i85bvjdfic9a965dy1d848
```